molgenis-app/src/main/java/org/molgenis/app/WebAppConfig.java

```diff
@@ -182,7 +186,8 @@ public void addUpgrades()
 		upgradeService.addUpgrade(new Step21SetLoggingEventBackend(dataSource));
 		upgradeService.addUpgrade(new Step22RemoveDiseaseMatcher(dataSource));
 		upgradeService.addUpgrade(step23RebuildElasticsearchIndex);
-		upgradeService.addUpgrade(new Step24UpdateApplicationSettings(dataSource, idGenerator));
+		upgradeService.addUpgrade(step24LanguagesPermissions);
+		upgradeService.addUpgrade(new Step25UpdateApplicationSettings(dataSource, idGenerator));
```

Comment:
if the order of these steps doesn't matter it is better to keep "UpdateApplicationSettings" as step 24 because it has already been merged